Dockerfile

```diff
@@ -4,9 +4,15 @@ WORKDIR /go/src/github.com/EFForg/starttls-backend
 
 RUN apt-get update && apt-get -y install postgresql-client
 
-ADD . .
+# Download vendorized dependencies
+ENV GO111MODULE=on
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
 
-RUN go get github.com/EFForg/starttls-backend
+# Build the binary
+ADD . .
```

Comment:
Using `COPY` instead of `ADD` is a [best practice when copying local files](https://stackoverflow.com/a/24958548). Let's take this opportunity to use it here :)